src/handlers.ts

```diff
@@ -375,71 +377,102 @@ export async function siteChangedWebhook(
     });
 }
 
+async function buildInboxItem(
```

Comment:
This was the guts of the `inboxHandler`. For now we want to use the same data structures in the newly added activities endpoint so have extracted this logic here and tidied up a little 🧹